modules/organizational-unit/locals.tf

```diff
@@ -0,0 +1,3 @@
+locals {
+  accounts = coalesce(var.accounts, {})
```

Comment:
This seems fine - and solves the problem, but maybe migration to newer terraform and setting proper object format would be a better way to go?